modules/tinymce/src/plugins/table/test/ts/browser/InsertTableWithColGroupsTest.ts

```diff
@@ -0,0 +1,71 @@
+import { Log, Logger, Pipeline, Step } from '@ephox/agar';
+import { UnitTest } from '@ephox/bedrock-client';
+import { TinyApis, TinyLoader } from '@ephox/mcagar';
+import Editor from 'tinymce/core/api/Editor';
+
```

Comment:
Why is there an empty new line here? I'd expect it before the editor import, not after. This is probably just a copy paste and is wrong elsewhere I guess.